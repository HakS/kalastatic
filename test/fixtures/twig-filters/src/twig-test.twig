---
title: Hello World!
---
<h1>{{ title }}</h1>
<p>{{ description }}</p>

<div class="twig-test">
  {{title|slugifythis}}
</div>

<h2 class="globalVar">{{ awesomeSauce }}</h2>
