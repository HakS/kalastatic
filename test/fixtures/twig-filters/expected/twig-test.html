<div class="twig-test">
  Hello-World
</div>

<h2 class="globalVar">This Is The AwesomeSauce!</h2>
